% Generated by roxygen2: do not edit by hand
% Please edit documentation in R/errors.R
\name{throw_validation_error}
\alias{throw_validation_error}
\title{Throw a validation error}
\usage{
throw_validation_error(
  message,
  parameter = NULL,
  value = NULL,
  call = sys.call(-1)
)
}
\arguments{
\item{message}{Error message}

\item{parameter}{Parameter name that failed validation}

\item{value}{The invalid value}

\item{call}{The call that generated the error (defaults to parent call)}
}
\description{
Throw a validation error
}
\keyword{internal}
